libnavigation-core/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     implementation project(':libnavigation-metrics')
     implementation dependenciesList.mapboxAndroidAccounts
     implementation dependenciesList.mapboxSdkTurf
+    implementation dependenciesList.mapboxSdkServices
```

Comment:
This is needed to get access to MapboxDirectionsRefresh

I'm personally a fan of putting all the API->Domain layer business logic in a module. Out of scope of this pull request though

Reply:
I tried again to put it there, but it didn't fit well. Router is mostly designed for HybridRouter. If we add something to Router it is unimplemented in offline. It introduces some other things, that could go one way or another

Reply:
moved everything so we can remove this dependency